<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DJ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>330</width>
      <height>353</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Puerto:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_Puertos"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="Boton_Actualizar_Puertos">
               <property name="text">
                <string>Actualizar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="Boton_Conectar">
             <property name="text">
              <string>Conectar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="Boton_Abrir">
                 <property name="text">
                  <string>Abrir</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Boton_Play">
                 <property name="text">
                  <string>Play</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Boton_Pause">
                 <property name="text">
                  <string>Pause</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Boton_Stop">
                 <property name="text">
                  <string>Stop</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QProgressBar" name="Progreso_Musica">
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="Boton_Mute">
               <property name="text">
                <string>Mute</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="Barra_Volumen">
               <property name="value">
                <number>10</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_Datos_Recibidos"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>362</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
